slam-ws/src/odom/params/odom.yaml

```diff
@@ -0,0 +1,258 @@
+imu: imu/data_raw
+
+imu_config: [true,  true,  false,
+               false, false, false,
+               false, false, false,
+               false, false, true,
+               false, false, false]
+
```

Comment:
this shouldn't exist here, this will most likely be removed once u merge the dev branch onto your branch